main.js

```diff
@@ -113,17 +137,18 @@ const askForSecrets = async () => {
 			});
 		});
 
-		dc.on('messageCreate', function (message) {
-			if (message.author.username !== dc.user.username) {
-				userInput(message.content, true, message);
-				if (dcUser == null || message.author.id !== dcUser.id) {
-					fs.writeFile(save, message.author.id, function (err) {
-						if (err) {
-							throw err;
-						}
-					});
-				}
-				dcUser = message.author;
+		dc.on('interactionCreate', async interaction => {
```

Comment:
regression:  you need  to check  if the user sending the command is authorized to administer the bot.